File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// N404ErrorHandler SDK exists test

require_once __DIR__ . '/../n404errorhandler_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = N404ErrorHandlerSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# N404ErrorHandler SDK exists test

require "minitest/autorun"
require_relative "../N404ErrorHandler_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = N404ErrorHandlerSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
